meson.build

```
@@ -716,6 +722,7 @@ if meson.version().version_compare('>=0.46.0')
       anyevent_i3,
       i3test_pm,
     ],
+    timeout: test_timeout,
```

I don't really think we need infinite timeout, we can just settle on a reasonable default since tests can hang.

Travis executes the tests in 20-30 seconds and it should be a reasonably slow machine. I think we can agree that after 120 seconds something is wrong.

```suggestion
    timeout: 120,
```

---

Seems reasonable.